db/migrations/202504231037_add_indexes.go

```diff
@@ -0,0 +1,45 @@
+package migrations
+
+import (
+	_ "embed"
+
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+var _202504231037_add_indexes = &gormigrate.Migration{
+	ID: "202504231037_add_indexes",
+	Migrate: func(db *gorm.DB) error {
+
+		if err := db.Transaction(func(tx *gorm.DB) error {
+
+			if err := tx.Exec(`
+DROP INDEX IF EXISTS idx_transactions_app_id;
+DROP INDEX IF EXISTS idx_transactions_request_event_id;
+DROP INDEX IF EXISTS idx_transactions_created_at;
+DROP INDEX IF EXISTS idx_transactions_settled_at;
+DROP INDEX IF EXISTS idx_transactions_app_id_type_state_created_at_settled_at_payment_hash;
```

Comment:
for example this one is removed, but what is it replaced with for list transactions?

Reply:
should this just be replaced with updated_at instead of `created_at_settled_at`?

Reply:
`idx_transactions_app_id_type_state_created_at_settled_at_payment_hash` is not used for `ListTransactions`, since there we never use payment hash, I think this was for `LookupInvoice` only

I added `idx_transactions_payment_hash_settled_at_created_at ` instead of this, since app_id doesn't differ much in the table and an index on these three should work better.

Reply:
> since app_id doesn't differ much in the table
@im-adithya what do you mean by this?

Reply:
But I saw you added the app id back in, thanks